### PR TITLE
Release prep for 1.2.1

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

**Comment by bastelfreak** [comment]
LGTM!

**Comment by DavidS** [comment]
@bastelfreak but does it install on 1.9 ?

File: lib/puppetlabs_spec_helper/rake_tasks.rb [hunk 1]
```diff
@@ -556,12 +555,15 @@ def max_thread_limit
   system("rake -T")
 end
 
-RuboCop::RakeTask.new(:rubocop) do |task|
-  # These make the rubocop experience maybe slightly less terrible
-  task.options = ['-D', '-S', '-E']
-end
-
-RuboCop::RakeTask.new("rubocop:auto_correct") do |task|
```
Comment (domcleal) [inline lib/puppetlabs_spec_helper/rake_tasks.rb]: Did you mean to delete this?

Comment (DavidS) [inline lib/puppetlabs_spec_helper/rake_tasks.rb]: yeah. rubocop rake tasks automatically add a :auto_correct task using the base task's options.

Comment (domcleal) [inline lib/puppetlabs_spec_helper/rake_tasks.rb]: Ah yes, I see it in rubocop now. Looks good!

**Comment by domcleal** [comment]
I've tested this against theforeman/foreman locally on Ruby 1.9.3 and it at least installs a compatible set of gems and begins to run `rake spec` successfully, without any problems introduced by rubocop deps.

**Comment by alexjfisher** [comment]
Works for me on ruby 1.9.3 too.

**Comment by logicminds** [comment]
Bundler uses different tag names (with v) than what this gem uses.   You may want to merge the rakefile from #166 as it had some custom tasks around that workflow and gemspec preview task.

**Comment by DavidS** [comment]
I don't see a reason for *not* using the current tooling's defaults. The
tags are not signed anyways.

**Pull request merged**